corehq/apps/app_manager/models.py

```diff
@@ -2141,23 +2143,23 @@ class CaseSearch(DocumentSchema):
     """
     Properties and search command label
     """
-    command_label = DictProperty(default={'en': 'Search All Cases'})
-    again_label = DictProperty(default={'en': 'Search Again'})
+    command_label = DictProperty(default={'en': 'Search All Cases'}, exclude_if_none=True)
+    again_label = DictProperty(default={'en': 'Search Again'}, exclude_if_none=True)
     search_label = SchemaProperty(CaseSearchLabel)
     search_again_label = SchemaProperty(CaseSearchAgainLabel)
     properties = SchemaListProperty(CaseSearchProperty)
     auto_launch = BooleanProperty(default=False)        # if true, skip the casedb case list
     default_search = BooleanProperty(default=False)     # if true, skip the search fields screen
     default_relevant = BooleanProperty(default=True)
-    additional_relevant = StringProperty()
-    search_filter = StringProperty()
-    search_button_display_condition = StringProperty()
+    additional_relevant = StringProperty(exclude_if_none=True)
+    search_filter = StringProperty(exclude_if_none=True)
+    search_button_display_condition = StringProperty(exclude_if_none=True)
     default_properties = SchemaListProperty(DefaultCaseSearchProperty)
-    blacklisted_owner_ids_expression = StringProperty()
-    additional_case_types = ListProperty(str)
-    data_registry = StringProperty()
-    data_registry_workflow = StringProperty()           # one of REGISTRY_WORKFLOW_*
-    additional_registry_cases = StringListProperty()  # list of xpath expressions
+    blacklisted_owner_ids_expression = StringProperty(exclude_if_none=True)
+    additional_case_types = ListProperty(str, exclude_if_none=True)
```

Comment:
not directly related: would be nice to use `StringListProperty` for consistency

Reply:
updated